CONTRIBUTING.md

```diff
@@ -38,3 +38,131 @@ are conflicts (even trivially resolvable ones).
+- A "Correct Patch" is one that satisfies the above requirements.
+
+### Development Process
+
+- Change on the project SHALL be governed by the pattern of accurately identifying problems and applying minimal, accurate solutions to these problems.
+- To request changes, a user SHOULD log an issue on the project Platform issue tracker.
+- The user or Contributor SHOULD write the issue by describing the problem they face or observe.
+- The user or Contributor SHOULD seek consensus on the accuracy of their observation, and the value of solving the problem.
+- Users SHALL NOT log feature requests, ideas, suggestions, or any solutions to problems that are not explicitly documented and provable.
+- Thus, the release history of the project SHALL be a list of meaningful issues logged and solved.
+- To work on an issue, a Contributor SHALL fork the project repository and then work on their forked repository.
+- To submit a patch, a Contributor SHALL create a Platform pull request back to the project.
+- A Contributor SHALL NOT commit changes directly to the project.
+- If the Platform implements pull requests as issues, a Contributor MAY directly send a pull request without logging a separate issue.
+- To discuss a patch, people MAY comment on the Platform pull request, on the commit, or elsewhere.
+- To accept or reject a patch, a Maintainer SHALL use the Platform interface.
+- Maintainers SHOULD NOT merge their own patches except in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
+- Maintainers SHALL NOT make value judgments on correct patches.
+- Maintainers SHALL merge correct patches from other Contributors rapidly.
+- The Contributor MAY tag an issue as "Ready" after making a pull request for the issue.
+- The user who created an issue SHOULD close the issue after checking the patch is successful.
+- Maintainers SHOULD ask for improvements to incorrect patches and SHOULD reject incorrect patches if the Contributor does not respond constructively.
+- Any Contributor who has value judgments on a correct patch SHOULD express these via their own patches.
+- Maintainers MAY commit changes to non-source documentation directly to the project.
+
+### Creating Stable Releases
+
+- The project SHALL have one branch ("master") that always holds the latest in-progress version and SHOULD always build.
+- The project SHALL NOT use topic branches for any reason. Personal forks MAY use topic branches.
+- To make a stable release someone SHALL fork the repository by copying it and thus become maintainer of this repository.
+- Forking a project for stabilization MAY be done unilaterally and without agreement of project maintainers.
+- A stabilization project SHOULD be maintained by the same process as the main project.
+- A patch to a stabilization project declared "stable" SHALL be accompanied by a reproducible test case.
```

Comment:
I don't understand that whole set about stable releases. Is it just an alternative to branching master for a release ? If so, it seems overcomplicated, and the only reason I can see for this is the ability for a non maintainer to apply any subsequent patches on that pseudo-branch, but we don't want some random doing so for obvious reasons.